standalone/standalone.py

```diff
@@ -97,6 +97,9 @@
 MERGE_SYSTEM_USER_MESSAGE = False
 FEW_SHOTS = 5
 BATCH_SIZE = 8
+JUDGE_CA_CERT_ENV_VAR_NAME = "JUDGE_CA_CERT_PATH"
+JUDGE_CA_CERT_PATH = "/tmp/judge_ca_cert.pem"
+JUDGE_CA_CERT_SECRET_KEY = "tls.crt"
```

Comment:
the key is usually one of 
- `ca.crt` 
- `service-ca.crt`
- `ca-bundle.crt` 

let's use one of them not to confuse the user